package.json

```diff
@@ -280,18 +282,17 @@
     "simple-progress-webpack-plugin": "^1.1.2",
     "sinon": "^1.17.1",
     "source-map-support": "^0.4.14",
+    "static-extend": "^0.1.2",
```

Comment:
This was also required in `devDeps` in https://github.com/artsy/reaction/pull/1399, looks like a bug somewhere. Will follow up.